pincer/middleware/interaction_create.py

```diff
@@ -41,23 +43,43 @@ def get_command_from_registry(interaction: Interaction):
         The command is not registered
     """
 
+    name: str = interaction.data.name
+    group = None
+    sub_group = None
+
+    options = interaction.data.options
+
+    if interaction.data.options:
+        option = options[0]
+        if option.type == 1:
+            group = name
+            name = option.name
+        elif option.type == 2:
+            group = interaction.data.name
+            sub_group = option.name
+            name = option.options[0].name
+
     with suppress(KeyError):
```

Comment:
isn't that same as using `ChatCommandHandler.register.get(...)` lol?

Reply:
If I did that it would have to be
```python
thing = ChatCommandHandler.register.get(...)
if thing:
    return thing
```
Im saving a line if that means anything